test/repository_test.rb

```diff
@@ -50,13 +51,17 @@
             persisted_user.age.must_equal(25)
           end
 
-          it 'assigns and persist created_at attribute' do
+          it 'assigns and persist created_at updated_at attribute' do
```

Comment:
good practices is to have 1 assertion per spec example, what do you think about splitting this test into 2?